package.json

```diff
@@ -300,7 +300,19 @@
         "title": "Move panel to bottom",
         "category": "PowerShell",
         "icon": "$(layout-panel-right)"
+      },
+      {
+        "title": "[PowerShell Debug Input Task] Wait for and return PSES startup process ID",
+        "command": "PowerShell.WaitForPsesActivationAndReturnProcessId",
+        "enablement": "false"
```

Comment:
What's "enablement" do? Does it prevent the extension from activating?

Reply:
@andyleejordan it simply hides it from someone trying to bind it to a key or something, you can't invoke it directly from the UI, only programatically.

Reply:
Cool, thanks. I bet there are others we should hide.